src/app/docs/dropdown/dropdown-docs.component.html

```diff
@@ -0,0 +1,330 @@
+
+  <sky-docs-design-guidelines>
+
+    <sky-docs-demo-page-section
+      heading="Usage"
+    >
+      <sky-docs-design-guideline
+        heading="Use when"
+        headingStyle="success"
+      >
+        <p>
+          Use dropdowns when users require access to multiple, related actions or when users require access to multiple, unrelated actions and you do not have room for multiple buttons.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/dropdown/usage-toolbar.png"
+          caption="use dropdowns to let users access multiple actions."
+          captionType="success"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+      <sky-docs-design-guideline>
+        <p>
+          Use context menu dropdowns for actions that affect individual items in a list.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/dropdown/usage-context-menu.png"
+          caption="use context menu dropdowns with items in a list."
+          captionType="success"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+      <sky-docs-design-guideline>
+        <p>
+          Use icon-only dropdowns when you do not have room for text labels. For example, use icon-only dropdowns in list toolbars when screen widths are narrow.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/dropdown/usage-icon.png"
+          caption="use icon-only dropdowns when there is not space to include a label."
+          captionType="success"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+      <sky-docs-design-guideline
+        heading="Don't use when"
+        headingStyle="danger"
+      >
+        <p>
+          Don't use dropdowns as form inputs. Use HTML <sky-code>&lt;select&gt;</sky-code> fields instead.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/dropdown/usage-icon.png"
+          caption="use dropdowns as form inputs."
+          captionType="danger"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Anatomy"
+    >
+      <sky-docs-anatomy
+        thumbnailPath="~/assets/img/guidelines/dropdown/anatomy.png"
+      >
+        <sky-docs-anatomy-item>Dropdown button</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>Dropdown menu</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>Action</sky-docs-anatomy-item>
+      </sky-docs-anatomy>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Options"
+    >
+      <sky-docs-design-guideline
+        heading="Button types"
+      >
+      <table class="sky-docs-table">
+          <tr>
```

Comment:
Check the number of spaces in this `table`'s tab spacing. There are a few places where you're using 4 spaces instead of 2.